src/hooks/useCart/useCart.tsx

```diff
@@ -233,6 +233,9 @@ export const useCart = (
       const transactions = Object.values(cart)
         .filter(({ quantity }) => quantity)
         .map(({ category, quantity, identifierInputs }) => {
+          for (const identifierInput of identifierInputs) {
```

Comment:
notice this additional loop to trim the identifier inputs would work, but we will be looping through them twice; once here, and another time at line 250. do you think we can validate and clean the identifier inputs together here?